planner/cascades/testdata/integration_suite_out.json

```diff
@@ -212,10 +212,13 @@
       {
         "SQL": "select max(a+b) from t",
         "Plan": [
-          "HashAgg_12 1.00 root funcs:max(Column#4)->Column#3",
-          "└─TableReader_13 1.00 root data:HashAgg_14",
-          "  └─HashAgg_14 1.00 cop[tikv] funcs:max(plus(test.t.a, test.t.b))->Column#4",
-          "    └─TableScan_10 10000.00 cop[tikv] table:t, range:[-inf,+inf], keep order:false, stats:pseudo"
+          "HashAgg_27 1.00 root funcs:max(Column#4)->Column#3",
+          "└─TopN_28 0.80 root Column#4:desc, offset:0, count:1",
+          "  └─TableReader_30 0.80 root data:TopN_31",
+          "    └─TopN_31 0.80 cop[tikv] Column#4:desc, offset:0, count:1",
+          "      └─Selection_33 0.80 cop[tikv] not(isnull(Column#4))",
+          "        └─HashAgg_34 1.00 cop[tikv] funcs:max(plus(test.t.a, test.t.b))->Column#4",
+          "          └─TableScan_25 10000.00 cop[tikv] table:t, range:[-inf,+inf], keep order:false, stats:pseudo"
```

Comment:
seems there is no beneficial when we eliminated the `max`/`min` but couldn't use index to retrieve the data orderly: both ways need to scan and process all the data in the query range.